locust/clients.py

```diff
@@ -215,7 +217,7 @@ class ResponseContextManager(LocustResponse):
     :py:meth:`failure <locust.clients.ResponseContextManager.failure>`.
     """
 
-    _manual_result = None
+    _manual_result: Optional[Union[bool, CatchResponseError, Exception]] = None
```

Comment:
CatchResponseError is a subtype of Exception, right? Does it still need to be specified?

Reply:
Ah yes, of course not.